notebooks/02_dj_webinar.ipynb

```diff
@@ -0,0 +1,1387 @@
+{
```

Comment:
<p>Here I would definitely say (not necessarily write) what the idea behind make and populate is: that the table itself knows how it can be filled. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/datajoint-webinar/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Agreed. I will mention that in the webinar.</p>